### PR TITLE
py sol for ac-nuske

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC: https://atcoder.jp/contests/agc015/submissions/58002179

File: solutions/silver/ac-nuske.mdx [hunk 1]
```diff
@@ -283,4 +283,76 @@ public class Main {
...
+			0 1
+			A new connected component forms!
+			"""
+			if not grid[i - 1][j] and not grid[i][j - 1]:
+				pref[i][j] += 1
+
+			"""
+              1
+            1 1
+            Two connected componets merge!
```
Comment (SansPapyrus683) [inline solutions/silver/ac-nuske.mdx:321]: bad formatting

Comment (freakin23) [reply solutions/silver/ac-nuske.mdx]: bot is to blame

Comment (freakin23) [reply solutions/silver/ac-nuske.mdx]: corrected it.

Comment (SansPapyrus683) [reply solutions/silver/ac-nuske.mdx]: > bot is to blame

this wouldn't happen if you used tabs to indent ur code like the guide convention states

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/silver/ac-nuske.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/silver/ac-nuske.mdx]: but i do use tabs

Comment (SansPapyrus683) [reply solutions/silver/ac-nuske.mdx]: a look at your initial commit shows that you clearly didn't

**Pull request merged**